package.json

```
@@ -1,5 +1,5 @@
 {
-  "name": "serverless-offline",
+  "name": "@tradecast/serverless-offline",
```

publishing to our `@tradecast` scope requires an publishConfig:
```suggestion
  "name": "@tradecast/serverless-offline",
  "publishConfig": {
        "always-auth": true,
        "registry": "https://npm.tradecast.eu:4879"
    },
```